### PR TITLE
Update Github issue template

### PR DESCRIPTION
**Pull request by TomAugspurger**
Mostly just using the `details` tag to reduce some clutter in all the issues, e.g.

---
#### A small, complete example of the issue

``` python
# Your code here
def fib(n):
    return fib(n-1) + fib(n-2)
```
#### Expected Output

Nothing
#### Output of `pd.show_versions()`

<details>
## INSTALLED VERSIONS

commit: None
python: 3.5.2.final.0
python-bits: 64
OS: Darwin
OS-release: 15.6.0
machine: x86_64
processor: i386
byteorder: little
LC_ALL: None
LANG: en_US.UTF-8
LOCALE: en_US.UTF-8

pandas: 0.19.0rc1+21.ge596cbf
nose: 1.3.7
pip: 8.1.2
setuptools: 26.1.1
Cython: 0.25a0
numpy: 1.11.1
scipy: 0.18.0
statsmodels: 0.8.0rc1
xarray: 0.7.2
IPython: 5.1.0
sphinx: 1.4.6
patsy: 0.4.1
dateutil: 2.5.3
pytz: 2016.6.1
blosc: 1.4.1
bottleneck: 1.0.0
tables: 3.2.2
numexpr: 2.6.1
matplotlib: 1.5.2
openpyxl: 2.3.5
xlrd: 0.9.4
xlwt: 1.0.0
xlsxwriter: None
lxml: 3.4.4
bs4: 4.4.1
html5lib: 0.9999999
httplib2: 0.9.2
apiclient: 1.5.1
sqlalchemy: 1.0.12
pymysql: 0.7.6.None
psycopg2: 2.6.2 (dt dec pq3 ext lo64)
jinja2: 2.8
boto: 2.39.0
pandas_datareader: None

</details>

### CODE REVIEW

**Comment by jorisvandenbossche** [comment]
+1 for the code snippet (indeed a lot of people don't do that)

What does the `<details>` do exactly? I don't see a difference in your top post (which uses it)

**Comment by jorisvandenbossche** [comment]
Ah, apparantly it is a firefox issue (in chrome it does work nicely! That's indeed very nice), but should be fixed in upcoming firefox (https://gist.github.com/ericclemmons/b146fe5da72ca1f706b2ef72a20ac39d)

Maybe we can adapt the details 'title' to indicate that the installed versions are in the details?

**Comment by TomAugspurger** [comment]
I edited the original to include a sample of what a new issue would look like.

**Comment by jreback** [comment]
lgtm

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -1,6 +1,15 @@
...
+```python
+# Your code here
+
+```
 
 #### Expected Output
 
-#### output of ``pd.show_versions()``
+#### Output of ``pd.show_versions()``
+
```
Comment (jreback) [inline .github/ISSUE_TEMPLATE.md]: shouldn't we also put:

```python
# output of ``pd.show_version()``

```

?

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -1,6 +1,15 @@
-#### Code Sample, a copy-pastable example if possible
+#### A small, complete example of the issue
```
Comment (jreback) [inline .github/ISSUE_TEMPLATE.md]: I think you also need ``#`` only (and not multiples), and this should be inside the quoting

**Review by jreback — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE.md (hunk 2 above)
Comment (jorisvandenbossche) [inline .github/ISSUE_TEMPLATE.md]: I think it is OK to leave it like this, and it gives a bit structure to the report with those titles (and by using a lot of `#`, the title is not too big)

Comment (TomAugspurger) [inline .github/ISSUE_TEMPLATE.md]: Yeah, the four hashes are because one `#` made the titles so large.

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (TomAugspurger) [inline .github/ISSUE_TEMPLATE.md]: Not sure what you mean. I just capitalized the 'o' here.

**Comment by jorisvandenbossche** [comment]
Firefox now also supports it! :-)

I am going to merge this, we can always refine later

**Pull request merged**